packages/jobs/src/core/Worker.ts

```diff
@@ -18,20 +18,48 @@ import { Executor } from './Executor.js'
 
 export interface WorkerOptions {
```

Comment:
I'm a little bit confused about the relation between this and `WorkerConfig`

Reply:
`WorkerConfig` is what you give to the `JobManager` when you're setting it up in `api/src/lib/jobs.ts`. `WorkerOptions` is what the actual `Worker` class requires on initialization. There are a couple of additional things that could be set internally by us when starting a worker.